frontend/src/time_utils.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Helper function to create Date from Timestamp model.
```

Comment:
nit: no need to add the "Helper function" comment - can just say Creates Date from Timestamp model.

and then the comment " // timestampDate.date.month is from 1-12, while JavaScript 'Date' is from 0-11." should be part of the function comment since it describes some assumption about both the input parameter and the return parameter format, which is quite important.

might also be good to speciify the date is at 00:00 .... instead of a specific time ?